openedx/core/lib/api/test_utils.py

```diff
@@ -27,6 +27,13 @@ def request_with_auth(self, method, *args, **kwargs):
         """Issue a get request to the given URI with the API key header"""
         return getattr(self.client, method)(*args, HTTP_X_EDX_API_KEY=TEST_API_KEY, **kwargs)
 
+    def request_without_auth(self, method, *args, **kwargs):
+        """
+        Issue a get request to the given URI without the API key header. This may be useful if you'll be calling
```

Comment:
I suppose this isn't only a "get" request and could be any really (e.g. post). :)